--- origsrc/xorg-server-1.3.0.0/Xi/stubs.c	2006-11-16 18:01:22.000000000 +0000
+++ src/xorg-server-1.3.0.0/Xi/stubs.c	2008-10-02 16:33:48.250000000 +0100
@@ -59,6 +59,10 @@
 #include <dix-config.h>
 #endif
 
+#ifdef HAVE_XWIN_CONFIG_H
+#include <xwin-config.h>
+#endif
+
 #include <X11/X.h>
 #include <X11/Xproto.h>
 #include "inputstr.h"
--- origsrc/xorg-server-1.3.0.0/fb/fbcmap.c	2006-09-18 07:04:17.000000000 +0100
+++ src/xorg-server-1.3.0.0/fb/fbcmap.c	2008-10-02 16:33:48.250000000 +0100
@@ -32,6 +32,10 @@
 #include <dix-config.h>
 #endif
 
+#ifdef HAVE_XWIN_CONFIG_H
+#include <xwin-config.h>
+#endif
+
 #include <X11/X.h>
 #include <X11/Xproto.h>
 #include "scrnintstr.h"
--- origsrc/xorg-server-1.3.0.0/mi/miinitext.c	2006-11-16 18:01:26.000000000 +0000
+++ src/xorg-server-1.3.0.0/mi/miinitext.c	2008-10-02 16:33:48.281250000 +0100
@@ -79,6 +79,11 @@
 #undef XF86VIDMODE
 #endif
 
+#ifdef HAVE_XWIN_CONFIG_H
+#include <xwin-config.h>
+#undef XFreeXDGA
+#endif
+
 #include "misc.h"
 #include "extension.h"
 #include "micmap.h"
