--- origsrc/xorg-server-1.5.2/Makefile.am	2008-10-10 13:12:48.000000000 -0500
+++ src/xorg-server-1.5.2/Makefile.am	2008-10-29 17:17:56.692163800 -0500
@@ -62,11 +62,13 @@
 	config \
 	hw
 
+if XORG
 aclocaldir = $(datadir)/aclocal
 aclocal_DATA = xorg-server.m4
 
 pkgconfigdir = $(libdir)/pkgconfig
 pkgconfig_DATA = xorg-server.pc
+endif
 
 EXTRA_DIST = xorg-server.pc.in xorg-server.m4 ChangeLog autogen.sh
 
