--- origsrc/xorg-server-1.5.2/glx/glxdriswrast.c	2008-10-10 13:12:48.000000000 -0500
+++ src/xorg-server-1.5.2/glx/glxdriswrast.c	2008-10-28 16:04:48.633516100 -0500
@@ -55,6 +55,10 @@
 #include "dispatch.h"
 #include "extension_string.h"
 
+#ifndef RTLD_LOCAL
+#define RTLD_LOCAL 0
+#endif
+
 typedef struct __GLXDRIscreen   __GLXDRIscreen;
 typedef struct __GLXDRIcontext  __GLXDRIcontext;
 typedef struct __GLXDRIdrawable __GLXDRIdrawable;
