--- origsrc/xorg-server-1.5.3/hw/xwin/winprocarg.c	2009-02-08 00:03:18.006996600 -0600
+++ src/xorg-server-1.5.3/hw/xwin/winprocarg.c	2009-02-08 18:50:07.373125000 -0600
@@ -670,16 +670,20 @@ ddxProcessArgument (int argc, char *argv
 	  /* Parameter is for all screens */
 	  for (j = 0; j < MAXSCREENS; j++)
 	    {
+#if 0
               if (!g_ScreenInfo[j].fMultiMonitorOverride)
                 g_ScreenInfo[j].fMultipleMonitors = TRUE;
+#endif
 	      g_ScreenInfo[j].fMWExtWM = TRUE;
 	    }
 	}
       else
 	{
 	  /* Parameter is for a single screen */
+#if 0
           if (!g_ScreenInfo[g_iLastScreen].fMultiMonitorOverride)
             g_ScreenInfo[g_iLastScreen].fMultipleMonitors = TRUE;
+#endif
 	  g_ScreenInfo[g_iLastScreen].fMWExtWM = TRUE;
 	}
 
@@ -699,8 +703,10 @@ ddxProcessArgument (int argc, char *argv
 	  /* Parameter is for all screens */
 	  for (j = 0; j < MAXSCREENS; j++)
 	    {
+#if 0
 	      if (!g_ScreenInfo[j].fMultiMonitorOverride)
 	        g_ScreenInfo[j].fMultipleMonitors = TRUE;
+#endif
 	      g_ScreenInfo[j].fMWExtWM = TRUE;
 	      g_ScreenInfo[j].fInternalWM = TRUE;
 	    }
@@ -708,8 +714,10 @@ ddxProcessArgument (int argc, char *argv
       else
 	{
 	  /* Parameter is for a single screen */
+#if 0
 	  if (!g_ScreenInfo[g_iLastScreen].fMultiMonitorOverride)
 	    g_ScreenInfo[g_iLastScreen].fMultipleMonitors = TRUE;
+#endif
 	  g_ScreenInfo[g_iLastScreen].fMWExtWM = TRUE;
 	  g_ScreenInfo[g_iLastScreen].fInternalWM = TRUE;
 	}
@@ -767,7 +775,7 @@ ddxProcessArgument (int argc, char *argv
 	  /* Parameter is for all screens */
 	  for (j = 0; j < MAXSCREENS; j++)
 	    {
-#if defined(XWIN_MULTIWINDOW) || defined(XWIN_MULTIWINDOWEXTWM)
+#if 0 // defined(XWIN_MULTIWINDOW) || defined(XWIN_MULTIWINDOWEXTWM)
               if (!g_ScreenInfo[j].fMultiMonitorOverride)
                 g_ScreenInfo[j].fMultipleMonitors = TRUE;
 #endif
@@ -777,7 +785,7 @@ ddxProcessArgument (int argc, char *argv
       else
 	{
 	  /* Parameter is for a single screen */
-#if defined(XWIN_MULTIWINDOW) || defined(XWIN_MULTIWINDOWEXTWM)
+#if 0 // defined(XWIN_MULTIWINDOW) || defined(XWIN_MULTIWINDOWEXTWM)
           if (!g_ScreenInfo[g_iLastScreen].fMultiMonitorOverride)
             g_ScreenInfo[g_iLastScreen].fMultipleMonitors = TRUE;
 #endif
