--- origsrc/xorg-server-1.5.3/hw/xwin/Makefile.am	2008-11-12 22:54:28.754097400 -0600
+++ src/xorg-server-1.5.3/hw/xwin/Makefile.am	2008-11-12 22:59:26.050972400 -0600
@@ -166,7 +166,8 @@
             $(XWINMODULES_CFLAGS) \
             -DXFree86Server
 
-dist_man1_MANS = XWin.man XWinrc.man
+dist_man1_MANS = XWin.man
+dist_man5_MANS = XWinrc.man
 
 EXTRA_DIST = \
 	_usr_X11R6_lib_X11_system.XWinrc \
--- origsrc/xorg-server-1.5.3/hw/xwin/XWin.man	2008-11-12 22:54:24.175972400 -0600
+++ src/xorg-server-1.5.3/hw/xwin/XWin.man	2008-11-12 23:32:42.536141000 -0600
@@ -49,13 +49,13 @@
 As it runs \fIXWin\fP writes messages indicating the most relevant events
 to  the console
 from which it was called and to a log file that by default is located at
-\fI/tmp/XWin.log\fP.  This file is mainly for debugging purposes.
+\fI__logdir__/XWin.log\fP.  This file is mainly for debugging purposes.
 
 
 .SH PREFERENCES FILE
 On startup \fIXWin\fP looks for the file \fI$HOME/.XWinrc\fP or, if
 the previous file does not exist,
-\fI/usr/X11R6/lib/X11/system.XWinrc\fP.  \fI.XWinrc\fP allows setting
+\fI__sysconfdir__/X11/system.XWinrc\fP.  \fI.XWinrc\fP allows setting
 preferences for the following:
 .br
 1- To include items into the menu associated with the \fIXWin\fP icon
@@ -155,7 +155,7 @@
 cursor.
 .TP 8
 .B "\-logfile \fIFile_Name\fP"
-Change the log file from the default located at \fI/tmp/XWin.log\fP to
+Change the log file from the default located at \fI__logdir__/XWin.log\fP to
 the one indicated by \fIFile_Name\fP.
 .TP 8
 .B "\-logverbose \fIlevel\fP"
@@ -245,7 +245,7 @@
 These options implement the xkeyboard extension for loading
 a particular keyboard map as the X server starts.  The behavior is similar
 to the \fIsetxkbmap\fP program.  The layout data is located at
-\fI/usr/X11R6/lib/X11/xkb/\fP.  Additional information is found in the
+\fI__datadir__/X11/xkb/\fP.  Additional information is found in the
 README files therein and in the man page of \fIsetxkbmap\fP.  For example
 in order to load a German layout for a pc105 keyboard one uses
 the options:
@@ -258,7 +258,7 @@
 
 
 .SH "SEE ALSO"
-X(__miscmansuffix__), Xserver(1), xdm(1), xinit(1), XWinrc(1), setxkbmap(1)
+X(__miscmansuffix__), Xserver(1), xdm(1), xinit(1), XWinrc(__filemansuffix__), setxkbmap(1)
 
 
 .SH BUGS
--- origsrc/xorg-server-1.5.3/hw/xwin/XWinrc.man	2008-11-12 22:54:21.707222400 -0600
+++ src/xorg-server-1.5.3/hw/xwin/XWinrc.man	2008-11-12 23:33:29.457715700 -0600
@@ -9,7 +9,7 @@
 The X Server for the X Window System on the Cygwin/X environment
 running on Microsoft Windows, \fIXWin\fP can be optionally configured
 with the \fIXWinrc\fP file.  A system-wide configuration file should
-be placed in \fI/usr/X11R6/lib/X11/system.XWinrc\fP, a per-user file
+be placed in \fI__sysconfdir__/X11/system.XWinrc\fP, a per-user file
 should be put at \fI$HOME/.XWinrc\fP.  The \fIsystem.XWinrc\fP file is
 read only if no \fI$HOME/.XWinrc\fP exist.
 .PP
