Mount options have changed in cygwin-1.7.

--- origsrc/xorg-server-1.6.0/hw/xwin/InitOutput.c	2009-03-12 19:25:14.953125000 -0500
+++ src/xorg-server-1.6.0/hw/xwin/InitOutput.c	2009-03-18 02:50:45.014352500 -0500
@@ -334,7 +334,7 @@ winCheckMount(void)
 
   while ((ent = getmntent(mnt)) != NULL)
   {
-    BOOL system = (strcmp(ent->mnt_type, "system") == 0);
+    BOOL system = (winCheckMntOpt(ent, "user") != NULL);
     BOOL root = (strcmp(ent->mnt_dir, "/") == 0);
     BOOL tmp = (strcmp(ent->mnt_dir, "/tmp") == 0);
     
@@ -361,7 +361,7 @@ winCheckMount(void)
       continue;
     level = curlevel;
 
-    if (winCheckMntOpt(ent, "binmode") == NULL)
+    if (winCheckMntOpt(ent, "binary") == NULL)
       binary = 0;
     else
       binary = 1;
