--- origsrc/xorg-server-1.6.0/hw/xwin/winclipboard.h	2009-02-25 13:12:12.000000000 -0600
+++ src/xorg-server-1.6.0/hw/xwin/winclipboard.h	2009-03-13 01:03:38.796875000 -0500
@@ -56,19 +56,8 @@
 #include "X11/Xutil.h"
 #include "X11/Xlocale.h"
 
-/* Fixups to prevent collisions between Windows and X headers */
-#define ATOM			DWORD
-
-#ifndef __CYGWIN__
-#define sleep(x) Sleep (1000 * (x))
-#endif
-
 /* Windows headers */
-#ifndef XFree86Server
-#define XFree86Server
-#endif
-#include <windows.h>
-#undef XFree86Server
+#include <X11/Xwindows.h>
 
 
 /* Clipboard module constants */
--- origsrc/xorg-server-1.6.0/hw/xwin/winclipboardthread.c	2009-03-12 19:25:02.921875000 -0500
+++ src/xorg-server-1.6.0/hw/xwin/winclipboardthread.c	2009-03-13 01:03:19.921875000 -0500
@@ -39,6 +39,7 @@
 #include <errno.h>
 #endif
 #include "X11/Xauth.h"
+#include <X11/Xlibint.h>
 
 
 /*
