--- origsrc/xorg-server-1.7.0.902/hw/xwin/winprefsyacc.y	2009-08-10 23:00:25.000000000 -0500
+++ src/xorg-server-1.7.0.902/hw/xwin/winprefsyacc.y	2009-10-20 15:14:43.672129000 -0500
@@ -32,6 +32,10 @@
  */
 /* $XFree86: $ */
 
+/* Conflict between a XWinrc token and an AC_DEFINE if debugging is enabled */
+#define DEBUG_TMP DEBUG
+#undef DEBUG
+
 #ifdef HAVE_XWIN_CONFIG_H
 #include <xwin-config.h>
 #endif
@@ -40,6 +44,11 @@
 #include <string.h>
 #include "winprefs.h"
 
+/* Conflict between a XWinrc token and an AC_DEFINE if debugging is enabled */
+#undef DEBUG
+#define DEBUG DEBUG_TMP
+#undef DEBUG_TMP
+
 /* The following give better error messages in bison at the cost of a few KB */
 #define YYERROR_VERBOSE 1
 
