--- origsrc/xorg-server-1.7.0.901/hw/xwin/winerror.c	2009-10-15 16:56:02.753543800 -0500
+++ src/xorg-server-1.7.0.901/hw/xwin/winerror.c	2009-10-15 23:37:40.124564400 -0500
@@ -141,6 +141,7 @@ winMessageBoxF (const char *pszError, UI
 	"Vendor: %s\n" \
 	"Release: %d.%d.%d.%d (%d)\n" \
 	"Contact: %s\n" \
+	"%s\n\n" \
 	"XWin was started with the following command-line:\n\n" \
 	"%s\n"
 
@@ -148,6 +149,7 @@ winMessageBoxF (const char *pszError, UI
 	   pszErrorF, VENDOR_STRING,
 		       XORG_VERSION_MAJOR, XORG_VERSION_MINOR, XORG_VERSION_PATCH, XORG_VERSION_SNAP, XORG_VERSION_CURRENT,
 		       VENDOR_CONTACT,
+		       BUILDERSTRING,
 	   g_pszCommandLine);
   if (!pszMsgBox)
     goto winMessageBoxF_Cleanup;
--- origsrc/xorg-server-1.7.3/hw/xwin/winprocarg.c	2009-10-15 16:56:09.563553200 -0500
+++ src/xorg-server-1.7.3/hw/xwin/winprocarg.c	2009-10-15 23:37:26.364545400 -0500
@@ -1552,6 +1552,7 @@ winLogVersionInfo (void)
   ErrorF ("Welcome to the XWin X Server\n");
   ErrorF ("Vendor: %s\n", VENDOR_STRING);
   ErrorF ("Release: %d.%d.%d.%d (%d)\n", XORG_VERSION_MAJOR, XORG_VERSION_MINOR, XORG_VERSION_PATCH, XORG_VERSION_SNAP, XORG_VERSION_CURRENT);
+  ErrorF ("%s\n\n", BUILDERSTRING);
   ErrorF ("Contact: %s\n", VENDOR_CONTACT);
 }
 
