--- origsrc/xorg-server-1.7.0.901/hw/xwin/winmultiwindowwindow.c	2009-10-16 20:53:42.390625000 +0100
+++ src/xorg-server-1.7.0.901/hw/xwin/winmultiwindowwindow.c	2009-10-18 20:11:24.250000000 +0100
@@ -580,7 +580,11 @@ winCreateWindowsWindow (WindowPtr pWin)
   winSelectIcons(pWin, &hIcon, &hIconSmall);
   if (hIcon) SendMessage (hWnd, WM_SETICON, ICON_BIG, (LPARAM) hIcon);
   if (hIconSmall) SendMessage (hWnd, WM_SETICON, ICON_SMALL, (LPARAM) hIconSmall);
- 
+
+  /* If we asked the native WM to place the window, synchronize the X window position */
+  if ((iX == CW_USEDEFAULT) || (iY == CW_USEDEFAULT))
+    winAdjustXWindow(pWin, hWnd);
+
   /* Change style back to popup, already placed... */
   SetWindowLongPtr(hWnd, GWL_STYLE, WS_POPUP | WS_CLIPCHILDREN | WS_CLIPSIBLINGS);
   SetWindowPos (hWnd, 0, 0, 0, 0, 0,

