diff -urN -x CYGWIN-PATCHES -x 'aclocal.m4*' -x ltmain.sh -x 'config.*' -x depcomp -x install-sh -x missing -x mkinstalldirs -x autom4te.cache -x '*compile' -x 'intltool*.in' -x 'xml-i18n-*.in' -x '*.pyc' -x '*.mo' -x '*.gmo' -x ABOUT-NLS -x Makevars.template -x COPYING -x INSTALL -x '*.orig' -x '*.rej' -x '*~' -x 'Makefile.in*' -x configure xorg-server-1.0.2-orig/Xext/fontcache.c xorg-server-1.0.2/Xext/fontcache.c
--- xorg-server-1.0.2-orig/Xext/fontcache.c	2005-07-03 03:53:36.000000000 -0500
+++ xorg-server-1.0.2/Xext/fontcache.c	2006-04-18 16:27:48.198000000 -0500
@@ -46,8 +46,8 @@
 #include "inputstr.h"
 #include "servermd.h"
 #define _FONTCACHE_SERVER_
-#include "fontcacheP.h"
-#include "fontcachstr.h"
+#include <X11/extensions/fontcacheP.h>
+#include <X11/extensions/fontcachstr.h>
 #include <X11/Xfuncproto.h>
 
 #include "swaprep.h"
diff -urN -x CYGWIN-PATCHES -x 'aclocal.m4*' -x ltmain.sh -x 'config.*' -x depcomp -x install-sh -x missing -x mkinstalldirs -x autom4te.cache -x '*compile' -x 'intltool*.in' -x 'xml-i18n-*.in' -x '*.pyc' -x '*.mo' -x '*.gmo' -x ABOUT-NLS -x Makevars.template -x COPYING -x INSTALL -x '*.orig' -x '*.rej' -x '*~' -x 'Makefile.in*' -x configure xorg-server-1.0.2-orig/configure.ac xorg-server-1.0.2/configure.ac
--- xorg-server-1.0.2-orig/configure.ac	2006-03-16 15:45:06.000000000 -0600
+++ xorg-server-1.0.2/configure.ac	2006-04-16 20:14:48.181125000 -0500
@@ -1143,7 +1143,7 @@
 if test "x$XPRINT" = xyes; then
 	PKG_CHECK_MODULES([XPRINT], [printproto x11 xfont $XDMCP_MODULES xau])
 	XPRINT_EXTENSIONS="$XEXT_LIB $DBE_LIB $XTRAP_LIB $RECORD_LIB $RENDER_LIB $COMPOSITE_LIB $RANDR_LIB $XI_LIB $FIXES_LIB $LBX_LIB $DAMAGE_LIB $XI_LIB $GLX_LIBS"
-	XPRINT_LIBS="$XPRINT_LIBS $DIX_LIB $XKB_LIB $XKB_STUB_LIB $XPRINT_EXTENSIONS $MI_LIB $MIEXT_DAMAGE_LIB $CWRAP_LIB $OS_LIB"
+	XPRINT_LIBS="$DIX_LIB $XKB_LIB $XKB_STUB_LIB $XPRINT_EXTENSIONS $MI_LIB $MIEXT_DAMAGE_LIB $CWRAP_LIB $OS_LIB $XPRINT_LIBS"
 	AC_SUBST([XPRINT_CFLAGS])
 	AC_SUBST([XPRINT_LIBS])
 
@@ -1164,7 +1164,7 @@
 		mingw*) XWIN="yes" ;;
 		*) XWIN="no" ;;
 	esac
-	XWIN_LIBS="$FB_LIB $MI_LIB $XEXT_LIB $XI_LIB $XKB_LIB $XKB_STUB_LIB $COMPOSITE_LIB $DAMAGE_LIB $LAYER_LIB $XPSTUBS_LIB $OS_LIB $SHADOW_LIB"
+	XWIN_LIBS="$FB_LIB $MI_LIB $XEXT_LIB $XI_LIB $RANDR_LIB $RENDER_LIB $XKB_LIB $XKB_STUB_LIB $COMPOSITE_LIB $DAMAGE_LIB $MIEXT_DAMAGE_LIB $MIEXT_SHADOW_LIB $MIEXT_LAYER_LIB $XPSTUBS_LIB $OS_LIB $LBX_LIB"
 	AC_SUBST([XWIN_LIBS])
 fi
 AC_MSG_RESULT([$XWIN])
diff -urN -x CYGWIN-PATCHES -x 'aclocal.m4*' -x ltmain.sh -x 'config.*' -x depcomp -x install-sh -x missing -x mkinstalldirs -x autom4te.cache -x '*compile' -x 'intltool*.in' -x 'xml-i18n-*.in' -x '*.pyc' -x '*.mo' -x '*.gmo' -x ABOUT-NLS -x Makevars.template -x COPYING -x INSTALL -x '*.orig' -x '*.rej' -x '*~' -x 'Makefile.in*' -x configure xorg-server-1.0.2-orig/hw/dmx/dmxinit.c xorg-server-1.0.2/hw/dmx/dmxinit.c
--- xorg-server-1.0.2-orig/hw/dmx/dmxinit.c	2005-07-11 19:52:47.000000000 -0500
+++ xorg-server-1.0.2/hw/dmx/dmxinit.c	2006-04-20 22:24:42.135250000 -0500
@@ -1060,3 +1060,8 @@
     return tp.tv_sec * 1000 + tp.tv_usec / 1000;
 }
 #endif
+
+void ddxBeforeReset(void)
+{
+    return;
+}
diff -urN -x CYGWIN-PATCHES -x 'aclocal.m4*' -x ltmain.sh -x 'config.*' -x depcomp -x install-sh -x missing -x mkinstalldirs -x autom4te.cache -x '*compile' -x 'intltool*.in' -x 'xml-i18n-*.in' -x '*.pyc' -x '*.mo' -x '*.gmo' -x ABOUT-NLS -x Makevars.template -x COPYING -x INSTALL -x '*.orig' -x '*.rej' -x '*~' -x 'Makefile.in*' -x configure xorg-server-1.0.2-orig/hw/xwin/InitInput.c xorg-server-1.0.2/hw/xwin/InitInput.c
--- xorg-server-1.0.2-orig/hw/xwin/InitInput.c	2005-07-04 17:10:43.000000000 -0500
+++ xorg-server-1.0.2/hw/xwin/InitInput.c	2006-04-16 19:59:40.865750000 -0500
@@ -32,9 +32,6 @@
 #include <xwin-config.h>
 #endif
 #include "win.h"
-#ifdef XWIN_CLIPBOARD
-# include "../../Xext/xf86miscproc.h"
-#endif
 #include "dixstruct.h"
 
 
diff -urN -x CYGWIN-PATCHES -x 'aclocal.m4*' -x ltmain.sh -x 'config.*' -x depcomp -x install-sh -x missing -x mkinstalldirs -x autom4te.cache -x '*compile' -x 'intltool*.in' -x 'xml-i18n-*.in' -x '*.pyc' -x '*.mo' -x '*.gmo' -x ABOUT-NLS -x Makevars.template -x COPYING -x INSTALL -x '*.orig' -x '*.rej' -x '*~' -x 'Makefile.in*' -x configure xorg-server-1.0.2-orig/hw/xwin/Makefile.am xorg-server-1.0.2/hw/xwin/Makefile.am
--- xorg-server-1.0.2-orig/hw/xwin/Makefile.am	2005-11-13 14:53:24.000000000 -0600
+++ xorg-server-1.0.2/hw/xwin/Makefile.am	2006-04-17 17:45:31.389375000 -0500
@@ -121,6 +121,7 @@
 	winpriv.h \
 	winresource.h \
 	winwindow.h \
+	$(top_srcdir)/Xi/stubs.c \
 	$(top_srcdir)/mi/miinitext.c \
 	$(top_srcdir)/fb/fbcmap.c \
 	$(SRCS_CLIPBOARD) \
@@ -151,7 +152,7 @@
              $(XWIN_LIBS) \
              $(XWINMODULES_LIBS)
 
-XWin_LDFLAGS = -mwindows -static
+XWin_LDFLAGS = -mwindows
 
 winprefsyacc.h: winprefsyacc.c
 winprefslex.c: winprefslex.l winprefsyacc.c winprefsyacc.h
diff -urN -x CYGWIN-PATCHES -x 'aclocal.m4*' -x ltmain.sh -x 'config.*' -x depcomp -x install-sh -x missing -x mkinstalldirs -x autom4te.cache -x '*compile' -x 'intltool*.in' -x 'xml-i18n-*.in' -x '*.pyc' -x '*.mo' -x '*.gmo' -x ABOUT-NLS -x Makevars.template -x COPYING -x INSTALL -x '*.orig' -x '*.rej' -x '*~' -x 'Makefile.in*' -x configure xorg-server-1.0.2-orig/hw/xwin/winmultiwindowwm.c xorg-server-1.0.2/hw/xwin/winmultiwindowwm.c
--- xorg-server-1.0.2-orig/hw/xwin/winmultiwindowwm.c	2005-07-05 13:25:44.000000000 -0500
+++ xorg-server-1.0.2/hw/xwin/winmultiwindowwm.c	2006-04-20 21:50:52.960250000 -0500
@@ -53,14 +53,7 @@
 #include <X11/cursorfont.h>
 
 /* Windows headers */
-#ifdef __CYGWIN__
-/* Fixups to prevent collisions between Windows and X headers */
-#define ATOM DWORD
-
-#include <windows.h>
-#else
-#include <Xwindows.h>
-#endif
+#include <X11/Xwindows.h>
 
 /* Local headers */
 #include "objbase.h"
